src/ntapfuse_ops.c

```diff
@@ -183,11 +182,40 @@ ntapfuse_chmod (const char *path, mode_t mode)
 int
 ntapfuse_chown (const char *path, uid_t uid, gid_t gid)
 {
-
     char fpath[PATH_MAX];
+    int res;
+    int usage = 0;
     fullpath (path, fpath);
+    //get file size
+    long fileSize = getFileSize (fpath);
+
+    //old user id for quota updating
+    int oldUid = getOwnerId (fpath);
```

Comment:
is it possible that getOwnerId call returns an error or invalid owner id? If so you can add some error handling here.